test/fixtures/temp-cache-fixture/a.js

```diff
@@ -0,0 +1,3 @@
+module.exports = function a() {
```

Comment:
Yeah, canceled before test suite was entirely done. Will remove them and ignore them as well. Thanks @sokra!